terraform/environments/corporate-staff-rostering/locals_security_groups.tf

```diff
@@ -116,28 +116,13 @@ locals {
           protocol    = -1
           self        = true
         }
-        # http135 = {
-        #   description = "Allow ingress from port 135"
-        #   from_port       = 135
-        #   to_port         = 135
-        #   protocol        = "Any"
-        #   cidr_blocks     = ["10.0.0.0/8"]
-        #   security_groups = []
-        # }
-        # http139 = {
-        #   description = "Allow ingress from port 139"
-        #   from_port       = 139
-        #   to_port         = 139
-        #   protocol        = "Any"
-        #   cidr_blocks     = ["10.0.0.0/8"]
-        #   security_groups = []
-        # }
-        https = {
-          description     = "Allow ingress from port 443"
-          from_port       = 443
-          to_port         = 443
+
+        dns = {
+          description     = "Allow ingress Azure domain controllers"
+          from_port       = 53
+          to_port         = 53
           protocol        = "TCP"
-          cidr_blocks     = ["10.0.0.0/8"]
+          cidr_blocks     = [for ip in module.ip_addresses.azure_fixngo_ips.devtest.domain_controllers : "${ip}/32"]
```

Comment:
This is a bit messy plus is devtest specific.  Better to add a local variable under security_group_cidrs_devtest and security_group_cidrs_preprod_prod like we do with other environment specific IP ranges.
You could also add the CIDRs into the ip_addresses module to avoid the for loop. 
 And then reference with module.ip_addresses.azure_fixngo_cidrs.devtest.domain_controllers